RecoLocalCalo/HcalRecAlgos/interface/HcalPedestalsGPU.h

```diff
@@ -0,0 +1,58 @@
+#ifndef RecoLocalCalo_HcalRecAlgos_interface_HcalPedestalsGPU_h
+#define RecoLocalCalo_HcalRecAlgos_interface_HcalPedestalsGPU_h
+
+#include "CondFormats/HcalObjects/interface/HcalPedestals.h"
+
+#ifndef __CUDACC__
+#include "HeterogeneousCore/CUDAUtilities/interface/HostAllocator.h"
+#include "HeterogeneousCore/CUDACore/interface/ESProduct.h"
+#endif
+
+class HcalPedestalsGPU {
+public:
+  struct Product {
+    ~Product();
+    float *values;
+    float *widths;
+    /*
```

Comment:
If these lines are not needed, they can be removed. Otherwise just add another comment line which explains why do you want to keep here even if  commented out.
(Same below for the same values)